### PR TITLE
Ensure all known issues are documented for Agent in 8.6.x

### PR DESCRIPTION
**Pull request by cmacknz**
- Add two known issues for 8.6.1 that are high severity and fixed in 8.6.2.
- Copy the release notes for a fix in Beats that applies to agent in 8.6.2 since it fixes a critical issue and needs to be visible.
- Add a known issue in 8.6.2 for the agent failing to install on MacOS Ventura. This technically applies to every version of the agent.

### CODE REVIEW

File: docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc [hunk 1]
```diff
@@ -155,6 +178,31 @@ To work around this problem, uninstall the {agent} and install it again with
 {fleet-server} enabled during the bootstrap process.
 ====
 
+[[known-issue-2232-8-6-1]]
+.Changing the {agent} log level can incorrectly disable inputs in supervised {beats}.
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc:182]:
```suggestion
.Changing the {agent} log level can incorrectly disable inputs in supervised {beats}
```

File: docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc [hunk 2]
```diff
@@ -155,6 +178,31 @@ To work around this problem, uninstall the {agent} and install it again with
...
+
+*Details*
+
+Data collection may be disabled when the {agent} log level is changed. Avoid changing the {agent} log level.
+
+Upgrade to 8.6.2 to fix the problem. For more information, refer to {agent-issue}2232[#2232].
+====
+
+[[known-issue-2328-8-6-1]]
+.{fleet-server} will crash when configured to use the Warning log level.
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc:194]:
```suggestion
.{fleet-server} will crash when configured to use the Warning log level
```

File: docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc [hunk 3]
```diff
@@ -75,6 +75,28 @@ To work around this problem, uninstall the {agent} and install it again with
 {fleet-server} enabled during the bootstrap process.
 ====
 
+[[known-issue-2103-8-6-2]]
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc:78]: We should make this entry and the one for 8.6.0 consistent. I like aspects of both descriptions, so I'll leave it to you to decide. (I'll wait for your update before I suggest edits to the description.)

File: docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc [hunk 4]
```diff
@@ -75,6 +75,28 @@ To work around this problem, uninstall the {agent} and install it again with
...
+.Installing {agent} on MacOS Ventura fails with a permissions error if Full Disk Access is not granted to the application used for installation.
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc:79]:
```suggestion
.Installing {agent} on MacOS Ventura fails with a permissions error if Full Disk Access is not granted to the application used for installation
```

Comment (dedemorton) [reply docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc:79]: If this is valid in 8.6.2, it should be added to 8.6.1, too. 

Incidentally I was able to install 8.6.1 successfully on Ventura so I thought this issue was resolved, but I guess it's still a problem? Maybe I fixed the issue myself without remembering. :-)

Comment (cmacknz) [reply docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc:79]: It is conditional on a few different things, including the version of MacOS Ventura. I should probably reword this to "can fail" and not "fails".

File: docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc (hunk 3 above)
Comment (cmacknz) [reply docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc]: Done, I somehow missed that this was already listed for 8.6.0. It applies to all versions.

File: docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc [hunk 5]
```diff
@@ -75,6 +75,21 @@ To work around this problem, uninstall the {agent} and install it again with
 {fleet-server} enabled during the bootstrap process.
 ====
 
+[[known-issue-issue-2103-8.6.2]]
+.Installating {agent} on MacOS Ventura may fail if Full Disk Access has not been granted to the application used for installation.
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc:79]:
```suggestion
.Installing {agent} on MacOS Ventura may fail if Full Disk Access has not been granted to the application used for installation.
```

File: docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc [hunk 6]
```diff
@@ -155,6 +171,46 @@ To work around this problem, uninstall the {agent} and install it again with
...
+*Details*
+
+{fleet-server} will crash when configured to use the Warning log level. Do not use the Warning log level.
+Affected {fleet-server} instances must be reinstalled to fix the problem.
+
+Upgrade to 8.6.2 to fix the problem. For more information, refer to {fleet-server-issue}2328[#2328].
+====
+
+[[known-issue-issue-2103-8.6.1]]
+.Installating {agent} on MacOS Ventura may fail if Full Disk Access has not been granted to the application used for installation.
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc:200]:
```suggestion
.Installing {agent} on MacOS Ventura may fail if Full Disk Access has not been granted to the application used for installation.
```

File: docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc [hunk 7]
```diff
@@ -255,21 +311,24 @@ For more information, refer to {agent-issue}2066[#2066].
 Be aware that the live query results shown in {kib} may be delayed by up to 5 minutes.
 ====
 
-[[known-issue-issue-2103]]
-.Installation of {agent} on MacOS Ventura may fail if Full Disk Access has not been granted to the installer
+[[known-issue-issue-2103-8.6.0]]
+.Installating {agent} on MacOS Ventura may fail if Full Disk Access has not been granted to the application used for installation.
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc:315]:
```suggestion
.Installing {agent} on MacOS Ventura may fail if Full Disk Access has not been granted to the application used for installation.
```

**Review by dedemorton — COMMENTED** [review]
Little typo fix. otherwise LGTM.

**Review by dedemorton — APPROVED** [review]
LGTM
